data/simple/bergamot/outputs/int8shiftAlphaAll/ssse3/quality_estimator.expected

```diff
@@ -0,0 +1,20 @@
+[src Sentence]:The Bergamot project will add and improve client-side machine translation in a web browser.
+[tgt Sentence]:Das Bergamot-Projekt wird die Übersetzung der Client-Seite der Maschine in einem Webbrowser ergänzen und verbessern.
+[score Sentence]:0
+[words Scores]:Das(0.000) Bergamot-Projekt(0.000) wird(0.000) die(0.000) Übersetzung(0.000) der(0.000) Client-Seite(0.000) der(0.000) Maschine(0.000) in(0.000) einem(0.000) Webbrowser(0.000) ergänzen(0.000) und(0.000) verbessern.(0.000)
+
+[src Sentence]:Unlike current cloud-based options, running directly on users’ machines empowers citizens to preserve their privacy and increases the uptake of language technologies in Europe in various sectors that require confidentiality.
+[tgt Sentence]:Im Gegensatz zu aktuellen Cloud-basierten Optionen, die direkt auf den Rechnern der Nutzer laufen, die Bürgerinnen und Bürger, ihre Privatsphäre zu bewahren und erhöht die Verbreitung von Sprachtechnologien in Europa in verschiedenen Sektoren, die Vertraulichkeit erfordern.
+[score Sentence]:0.000
+[words Scores]:Im(0.000) Gegensatz(0.000) zu(0.000) aktuellen(0.000) Cloud-basierten(0.000) Optionen,(0.000) die(0.000) direkt(0.000) auf(0.000) den(0.000) Rechnern(0.000) der(0.000) Nutzer(0.000) laufen,(0.000) die(0.000) Bürgerinnen(0.000) und(0.000) Bürger,(0.000) ihre(0.000) Privatsphäre(0.000) zu(0.000) bewahren(0.000) und(0.000) erhöht(0.000) die(0.000) Verbreitung(0.000) von(0.000) Sprachtechnologien(0.000) in(0.000) Europa(0.000) in(0.000) verschiedenen(0.000) Sektoren,(0.000) die(0.000) Vertraulichkeit(0.000) erfordern.(0.000)
```

Comment:
Treat numeric output and text output differently (don't intersperse both in output) so that when floating-point rounding off varies (if at all) they're approx compared with some tolerance threshold (absolute string match or relaxed "string" match would fail). Have one output printing the ByteRanges and another printing the numbers.